packages/hooks/src/use-local-storage.tsx

```
@@ -1,8 +1,7 @@
 import { useState, useEffect, useCallback } from "react";
 
 export const getValueFromLocalStorage = (key: string, defaultValue: any) => {
-  if (typeof window === undefined || typeof window === "undefined")
-    return defaultValue;
+  if (typeof window === undefined || typeof window === "undefined") return defaultValue;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal
Unsafe fix: Compare the result of `typeof` with a valid type name


(lint/suspicious/useValidTypeof)